keycloak/keycloak-admin-ui
# Add Save/Cancel buttons and logic to LDAP settings

## Motivation
https://issues.redhat.com/browse/APPDUX-720

## Brief Description
Adds the ability to save (or cancel) changes made to the User Federation Kerberos settings page.

## Verification Steps
1. Go to User federation and select an LDAP card.
2. Change any of the values in any of the subsections.
3. Click Save. You should see a success alert.
4. Verify that the changes have been saved by traversing anywhere else in the UI and then re-clicking the card to make sure the values that were changed are displayed correctly.

## Checklist:

## Additional Notes
